pom.xml

```diff
@@ -0,0 +1,51 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>de.mapoll</groupId>
+
+	<artifactId>fritzTR064-client</artifactId>
```

Comment:
This deviates from the project name; but since it's "just" a client, as opposed to other potential use cases, I felt this distinction makes sense.